test/integration/targets/ecs_tag/tasks/main.yml

```diff
@@ -0,0 +1,376 @@
+- block:
+    - name: set up aws connection info
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{ aws_access_key }}"
+          aws_secret_key: "{{ aws_secret_key }}"
+          security_token: "{{ security_token }}"
+          region: "{{ aws_region }}"
+      no_log: True
+
+    - name: create ecs cluster
+      ecs_cluster:
+        name: "{{ resource_prefix }}"
+        state: present
+        <<: *aws_connection_info
```

Comment:
```suggestion
```
Once module_defaults is added, remove the yaml anchor from each task that uses it.